game/plugins/src/main/kotlin/gg/rsmod/plugins/content/skills/thieving/pickpocket/PickpocketNpc.kt

```diff
@@ -0,0 +1,462 @@
+package gg.rsmod.plugins.content.skills.thieving.pickpocket
+
+import gg.rsmod.game.model.weight.impl.WeightItem
+import gg.rsmod.game.model.weight.impl.WeightItemSet
+import gg.rsmod.plugins.api.cfg.Items
+import gg.rsmod.plugins.api.cfg.Npcs
+
+//Rarity weighting
+private const val ALWAYS = 0
+private const val COMMON = 256
+private const val UNCOMMON = 32
+private const val RARE = 8
+private const val VERY_RARE = 1
```

Comment:
These should be inside the PickpocketNpcs enum class, otherwise they will be globally exposed to other files. Should be defined inside the companion object as they are constants